src/ActionModal/components/DelegationModal.vue

```diff
@@ -399,6 +406,34 @@ export default {
       update(data) {
         return data.balance || { amount: 0 }
       }
+    },
+    totalStaked: {
+      query: gql`
+        query overview($networkId: String!, $address: String!) {
+          overview(networkId: $networkId, address: $address) {
+            liquidStake
+            totalStake
+          }
+        }
+      `,
+      skip() {
+        /* istanbul ignore next */
+        return (
+          (!this.address || !this.network) &&
+          // only needed for polkadot to determine if user needs to set an amount
+          this.network !== "polkadot-testnet"
```

Comment:
Doesn't make more sense to do `!this.network.startsWith("polkadot")`?

Reply:
sure! let'S adjust once we have polkadot mainnet?

Reply:
```suggestion
         !this.network.startsWith("polkadot")
```